aws/table_aws_ssm_managed_instance_compliance.go

```diff
@@ -0,0 +1,158 @@
+			},
+			{
+				Name:        "status",
+				Description: "The status of the compliance item.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "compliance_type",
+				Description: "The compliance type.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "resource_type",
+				Description: "The type of resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "severity",
+				Description: "The severity of the compliance status.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "details",
+				Description: "A key-value combination details for the compliance item.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "execution_summary",
+				Description: "A summary for the compliance item.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.From(ssmManagedInstanceComplianceTitle),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getSSMManagedInstanceComplianceAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listSsmManagedInstanceCompliances(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listSsmManagedInstanceCompliances", "AWS_REGION", region)
+
+	// Create session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+	instanceId := h.Item.(*ssm.InstanceInformation).InstanceId
+
+	// Build the params
+	params := &ssm.ListComplianceItemsInput{
+		ResourceIds: []*string{instanceId},
+	}
+
+	// List call
+	data, err := svc.ListComplianceItems(params)
+	if err != nil {
+		if a, ok := err.(awserr.Error); ok {
+			if a.Code() == "InvalidResourceId" || a.Code() == "ValidationException" {
+				return nil, nil
+			}
+			return nil, err
+		}
+	}
+	for _, item := range data.ComplianceItems {
+		d.StreamListItem(ctx, item)
+	}
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getSSMManagedInstanceComplianceAkas(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getSSMInstanceComplianceAkas")
+	data := h.Item.(*ssm.ComplianceItem)
+
+	commonData, err := getCommonColumns(ctx, d, h)
+	if err != nil {
+		return nil, err
+	}
+	commonColumnData := commonData.(*awsCommonColumnData)
+
+	akas := []string{"arn:" + commonColumnData.Partition + ":ssm:" + commonColumnData.Region + ":" + commonColumnData.AccountId + ":managed-instance-compliance/" + *data.Id}
```

Comment:
@bigdatasourav Can you please switch the ARN format to use: `arn:${Partition}:ssm:${Region}:${Account}:managed-instance/${InstanceId}/compliance-item/${ComplianceItemId}:${ComplianceItemType}`? This format is more consistent with AWS' other SSM ARNs, and includes the 3 pieces of information that are unique to each row (compliance ID, compliance type, and resource ID).